sdk/communication/azure-communication-callingserver/swagger/README.md

```diff
@@ -8,7 +8,7 @@ To build the SDK for Server Calling Client, simply Install AutoRest and in this
 ### Setup
 ```ps
 Fork and clone https://github.com/Azure/autorest.java
-git checkout v4
+git checkout main
```

Comment:
This may be out of scope for this PR but we should stop using source builds for generating code. We should use one of the [github release versions](https://github.com/Azure/autorest.java/releases) instead.